numpy/random/mtrand/mtrand.pyx

```diff
@@ -713,10 +735,11 @@ cdef class RandomState:
 
         Parameters
         ----------
-        state : tuple(str, ndarray of 624 uints, int, int, float)
+        state : tuple(tuple(int, str), ndarray of 624 uints, int, int, float)
             The `state` tuple has the following items:
 
-            1. the string 'MT19937', specifying the Mersenne Twister algorithm.
+            1. the version of the RNG methods, and the string 'MT19937',
+               specifying the Mersenne Twister algorithm.
```

Comment:
None of the options are awesome, but that's just the nature of pickle tuples for you :-). Let's just put the version first: `(version, "MT19937", ...)`. Easy to check for, since old-style tuples start with a string and new-style ones will start with an int.